nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/excel/RowIterator.java

```diff
@@ -129,4 +151,37 @@ private boolean hasExhaustedRows() {
         }
         return exhausted;
     }
+
+    private boolean isXSSFExcelFile(InputStream in, final String password) {
```

Comment:
Minor naming recommend:
```suggestion
    private boolean isXmlFormat(final InputStream in, final String password) {
```

Reply:
or perhaps
```suggestion
    private boolean isXSSF(InputStream in, final String password) {
```